### PR TITLE
Copter: added MOT_BOOST_SCALE

### PR DESCRIPTION
**Pull request by tridge**
This adds support for booster motors on multicopters.

https://goo.gl/photos/Gcv6xonuQeabdVqp8

https://www.youtube.com/watch?v=3enY4NVHRXA&list=LLJzpx7cFPmzIW1RbGpZesYA&index=3

### CODE REVIEW

File: libraries/AP_Motors/AP_MotorsMulticopter.cpp [hunk 1]
```diff
@@ -162,6 +162,14 @@ const AP_Param::GroupInfo AP_MotorsMulticopter::var_info[] = {
     // @Increment: 0.1
     // @User: Advanced
     AP_GROUPINFO("SPOOL_TIME",   36, AP_MotorsMulticopter,  _spool_up_time, AP_MOTORS_SPOOL_UP_TIME_DEFAULT),
+
+    // @Param: BOOST_SCALE
+    // @DisplayName: Motor boost scale
+    // @Description: This is a scaling factor for vehicles with a vertical booster motor used for extra lift. It is used with electric multicopters that have an interal combusion booster motor for longter endurance. The output to tke BoostThrottle servo function is set to the current motor thottle times this scaling factor. A higher scaling factor will put more of the load on the booster motor. A value of 1 will set the BoostThrottle equal to the main throttle.
```
Comment (OXINARF) [inline libraries/AP_Motors/AP_MotorsMulticopter.cpp]: The output to **tke** BoostThrottle -> The output to **the** BoostThrottle

Comment (tridge) [inline libraries/AP_Motors/AP_MotorsMulticopter.cpp]: fixed, thanks!

**Comment by urenaf122** [comment]
Great project 
How do you connect the gas combustion engine to the pixhawk?  

You place a servo to the gas engine throttle and then how do you connect it to the pixhawk  ???

What are the steps and electronic components ?

**Comment by rmackay9** [comment]
@urenaf122,

These questions should really go into [the support forums](https://discuss.ardupilot.org/c/arducopter/copter-40/155) but in any case we have some instructions here on the wiki: https://ardupilot.org/copter/docs/booster-motor.html

File: libraries/AP_Motors/AP_MotorsMulticopter.cpp (hunk 1 above)
Comment (urenaf122) [reply libraries/AP_Motors/AP_MotorsMulticopter.cpp]: https://youtu.be/6c5m10Wtuow

Hello 
How do you connect the  PWM to the pixhawk to the servo that connects to the Gad Combustion Engine ?

https://youtu.be/6c5m10Wtuow

**Comment by urenaf122** [comment]
Thanks 

Please see this brief link

https://youtu.be/6c5m10Wtuow

**Comment by urenaf122** [comment]
Sorry
I will re-submit properly  later

**Pull request merged**